internal/mode/static/provisioner/objects_test.go

```diff
@@ -415,14 +446,32 @@ func TestBuildNginxResourceObjects_DockerSecrets(t *testing.T) {
 		},
 		Data: map[string][]byte{"data": []byte("docker")},
 	}
-	fakeClient := fake.NewFakeClient(dockerSecret)
+
+	dockerSecretTeaName := dockerTestSecretName + "-Tea"
```

Comment:
Just to prevent confusion, I wouldn't use `tea` and `chai` in the docker secret names. Since these secrets are for pulling nginx images, and we typically use these names for routes.

You could just use something like `registry1` and `registry2`